src/test/kotlin/com/zepben/evolve/services/network/testdata/LvEquipmentBelowFeederHeadNetwork.kt

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2022 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.zepben.evolve.services.network.testdata
+
+import com.zepben.evolve.cim.iec61970.base.core.BaseVoltage
+import com.zepben.evolve.services.network.NetworkService
+import com.zepben.evolve.testing.TestNetworkBuilder
+
+object LvEquipmentBelowFeederHeadNetwork {
+
+    //
+    // - or |: LV line
+    // = or #: HV line
+    //
+    //      c1     c3     c7         c9     c10
+    // b0 ======+======+====== tx8 ------+======
+    //          |      #
+    //       c2 |   c4 #
+    //          |      #
+    //                tx5
+    //                 |
+    //              c6 |
+    //                 |
+    //
+    // fdr11 head terminal is b0-t2
+    //
+    fun create(): NetworkService {
```

Comment:
suggestion: Could be reworked in the same way as `HvEquipmentBelowLvFeederHeadNetwork`